spec/send_text_spec.rb

```diff
@@ -0,0 +1,5 @@
+describe SendText do
+  it { is_expected.to respond_to(:go_send) }
```

Comment:
For testing the third part API, there's some good reading in the takeaway challenge review doc that you can see here: https://github.com/Maldorana/takeaway-challenge/blob/main/docs/review.md#step-2-tests-and-_specrb-files

You only need to go into as much detail as is in the review doc for this - so testing that the correct methods are called, but not actually calling the API each time the test is run.